### DISCUSSION TITLE
Astro Dev Toolbar Integration

### DISCUSSION THREAD

**Discussion by SSmale** [withastro/docs#6256] [Ideas]
I was reviewing the dev toolbar docs and they are quite hard to follow as some of the information is in that page, some more is on the Integrations page and then the more complex stuff is on the Vite docs. It might be an idea to have an example for common use cases. 

For consideration for that example. I created [this toolbar app](https://github.com/SSmale/astro-paper/commit/fe4229be932abc30b880213ae64eb0c650e9be8f) for the [AstroPaper Blog project](https://github.com/satnaing/astro-paper) to allow you to toggle the ability to see DRAFT posts. 

It has server to client comms over websocket, semi dynamic tooltip and icon and server client state syncing. 

https://github.com/withastro/docs/assets/17761689/b3f344f8-81a0-4594-8cab-0829939be4bd

**Comment by sarah11918** [comment]
Hi! This would be great to post in our Discord `#showcase` channel! If you're on Discord, please do share it with the community there since I'm sure people would love to see it!

In fact, I was just coming to edit our Discussion categories and remove this one as the Docs Discussion isn't really a place where people will be looking for "Show and Tell" type stuff. 😅 

The best place here in Discussions to share an idea for new content is in the "Idea" section, so I'm going to close this in this category (and delete the category entirely!) but do feel free to share it on Discord and/or start a discussion in the Ideas section here! 🙌
